test/gtest/group_conv.hpp

```diff
@@ -497,20 +525,26 @@ std::vector<miopenTensorLayout_t> GetLayoutValues()
 
 } // namespace group_conv
 
-#define DEFINE_GROUP_CONV_TEST(ndim, type, dir)                                             \
-    struct GroupConv##ndim##D_##dir##_##type : GroupConvTestFix<ndim, type, Direction::dir> \
-    {                                                                                       \
-    };                                                                                      \
-    TEST_P(GroupConv##ndim##D_##dir##_##type, GroupConv##ndim##D_##dir##_##type##_Test)     \
-    {                                                                                       \
-        RunSolver();                                                                        \
-    }                                                                                       \
-    INSTANTIATE_TEST_SUITE_P(                                                               \
-        GroupConv##ndim##D_##dir##_##type##_Suite,                                          \
-        GroupConv##ndim##D_##dir##_##type,                                                  \
-        testing::Combine(                                                                   \
-            testing::ValuesIn(GroupConvTestConfig<ndim>::GetConfigs<Direction::dir>()),     \
+#define DEFINE_GROUP_CONV_TEST(ndim, alpha, beta, type, dir, ab_case)                   \
+    struct GroupConv##ndim##D_##dir##_##type##_##ab_case                                \
+        : GroupConvTestFix<ndim, type, Direction::dir>                                  \
+    {                                                                                   \
+    };                                                                                  \
+    TEST_P(GroupConv##ndim##D_##dir##_##type##_##ab_case,                               \
+           GroupConv##ndim##D_##dir##_##type##_##ab_case##_Test)                        \
+    {                                                                                   \
+        RunSolver();                                                                    \
+    }                                                                                   \
+    INSTANTIATE_TEST_SUITE_P(                                                           \
+        GroupConv##ndim##D_##dir##_##type##_##ab_case##_Suite,                          \
+        GroupConv##ndim##D_##dir##_##type##_##ab_case,                                  \
+        testing::Combine(                                                               \
+            testing::ValuesIn(GroupConvTestConfig<ndim>::GetConfigs<Direction::dir>()), \
+            testing::ValuesIn({alpha}),                                                 \
```

Comment:
You can put a few values of `alpha` and `beta` here that cover all the cases and then you don't have to modify any of the `.cpp` test files or macros below.